scripts/kokoro-client.sh

```
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Fast Kokoro TTS client - talks to persistent server
+# Usage: kokoro-client.sh "text" /output/path.mp3
+export LD_LIBRARY_PATH=/home/clawbox/.local/lib:/home/clawbox/.local/lib/python3.10/site-packages/nvidia/cusparselt/lib:/usr/local/cuda/lib64:${LD_LIBRARY_PATH}
+
+TEXT="$1"
+OUTPUT="$2"
+SOCKET="/tmp/kokoro-server.sock"
+TMPWAV=$(mktemp /tmp/kokoro_XXXXXX.wav)
+trap "rm -f $TMPWAV" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)